keps/sig-node/runtime-class-scheduling.md

```diff
@@ -190,43 +181,26 @@ inculde:
   RuntimeClasses would need a toleration to enable them to be run on those
   nodes.
 
-### RuntimeController
+### Scheduler
 
-RuntimeController is a new in-tree admission plugin that should eventually be
-enabled on almost every Kubernetes clusters. The role of the controller for
-scheduling is to merge the topology constraints from the RuntimeClass into the
-PodSpec. Eventually, the controller's responsibilities may grow, such as to
-merge in [pod overhead][] or validate feature compatibility.
+A new scheduler predicate will manage the RuntimeClass topology. It will lookup
```

Comment:
Maybe we should discuss predicate composition vs adding support of runtimeClass into other existing predicate

Reply:
If we add it to existing predicates, we don't get a useful error message when the pod can't be scheduled, right?

Reply:
Yes

Reply:
@tallclair - if we merge tolerations into podSpec how can we inform user correctly (UX) when not finding a node that fits ?

Reply:
I don't think that changes anything, as the tolerations would be resolved by the same `PodToleratesNodeTaints` predicate? But I might be misunderstanding how the debug messages are created.

Reply:
Do you mean that we’d still check the runtimeClass when computing `podToleratesNodeTaints` ?

Reply:
I think we can move forward, and discuss this while implementing